packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -587,3 +615,176 @@ describe('toAuditOutputs', () => {
+    await expect(getBudgets()).resolves.toStrictEqual([]);
+  });
+
+  it('should load budgets from path is specified', async () => {
+    const budgets: Budget[] = [
+      {
+        path: '*',
+        resourceCounts: [
+          {
+            budget: 3,
+            resourceType: 'media',
+          },
+        ],
+      },
+    ];
+    vol.fromJSON(
+      {
+        'lh-budgets.json': JSON.stringify(budgets, null, 2),
+      },
+      MEMFS_VOLUME,
+    );
+    await expect(getBudgets('lh-budgets.json')).resolves.toEqual(budgets);
+  });
+
+  it('should throw if path is specified wrong', async () => {
+    await expect(getBudgets('wrong.xyz')).rejects.toThrow(
+      'ENOENT: no such file or directory',
+    );
+  });
+});
+
+describe('setLogLevel', () => {
+  const debugLib = debug as { enabled: (flag: string) => boolean };
+  beforeEach(() => {
+    log.setLevel('info');
+  });
+
+  /**
+   *
+   *  case 'silent':
+   *    debug.enable('-LH:*');
+   *    break;
+   *  case 'verbose':
+   *    debug.enable('LH:*');
+   *    break;
+   *  case 'warn':
+   *    debug.enable('-LH:*, LH:*:warn, LH:*:error');
+   *    break;
+   *  case 'error':
+   *    debug.enable('-LH:*, LH:*:error');
+   *    break;
+   *  default: // 'info'
+   *    debug.enable('LH:*, -LH:*:verbose');
+   */
+
+  it('should set log level to info if no options are given', () => {
+    setLogLevel();
+    expect(log.isVerbose()).toBe(false);
+    expect(debugLib.enabled('LH:*')).toBe(true);
+    expect(debugLib.enabled('LH:*:verbose')).toBe(false);
+  });
+
+  it('should set log level to verbose', () => {
+    setLogLevel({ verbose: true });
+    expect(log.isVerbose()).toBe(true);
+    expect(debugLib.enabled('LH:*')).toBe(true);
+    expect(debugLib.enabled('LH:*:verbose')).toBe(false);
+  });
+
+  it('should set log level to quiet', () => {
+    setLogLevel({ quiet: true });
+    expect(log.isVerbose()).toBe(false);
+    expect(debugLib.enabled('LH:*')).toBe(true);
+    expect(debugLib.enabled('-LH:*')).toBe(true);
+    expect(debugLib.enabled('LH:*:verbose')).toBe(false);
+  });
+
+  it('should set log level to verbose if verbose and quiet are given', () => {
+    setLogLevel({ verbose: true, quiet: true });
+    expect(log.isVerbose()).toBe(true);
+    expect(debugLib.enabled('LH:*')).toBe(true);
+    expect(debugLib.enabled('LH:*:verbose')).toBe(false);
+  });
+});
+
+describe('validateFlags', () => {
+  it('should work with empty flags', () => {
+    expect(validateFlags()).toStrictEqual({});
+  });
+
+  it('should forward supported entries', () => {
+    expect(validateFlags({ verbose: true })).toStrictEqual({ verbose: true });
+  });
+
+  it('should remove unsupported entries and log', () => {
+    expect(
+      validateFlags({ 'list-all-audits': true, verbose: true } as Flags),
+    ).toStrictEqual({ verbose: true });
+    expect(getLogMessages(ui().logger).at(0)).toBe(
+      '[ blue(info) ] ⚠ The following used flags are not supported: list-all-audits',
```

Comment:
Good question 😂

Reply:
Can I change something to have a more intuitive read of the test?

Reply:
I think that testing that a specified level is correctly displayed belongs in the logger library integration tests. In the usage of the logger elsewhere, one should then assume the levels are correct and only focus on the content. So I would skip the formatting and focus on the values the test is trying to verify.

Reply:
I had cleanup logic in the getMessages helper before but and later when merging changes from main I saw @matejchalk also tested the style and I broke his test. 

So I added it here https://github.com/code-pushup/cli/issues/552. If this gets implemented it will break this test and matej's test and we can decide if we implement it in a way where general styles for the log level and styles in text as bold, or color's in the respective test or if we remove it all together. In any case it is a little change.

Reply:
I am okay with either. If this is the only thing remaining to merge the PR, I would not wait to change this.
If we go for A, please mention this refactor in the issue 🙏

Reply:
Added it to PR under "Follow up PR"